### PR TITLE
Commit flow

### PR DESCRIPTION
**Pull request by huwshimi**
Set up the deployment flow for when you are committing to an existing model.

### CODE REVIEW

**Comment by huwshimi** [comment]
QA:
- add an application
- open the deployment flow
- you should see the could, credentials and region are already set and are not changeable
- at the bottom you should see a list of changes

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/5783/
Test PASSed.

File: jujugui/static/gui/src/app/app.js [hunk 1]
```diff
@@ -1139,12 +1139,21 @@ YUI.add('juju-gui', function(Y) {
...
+      }
       ReactDOM.render(
         <window.juju.components.DeploymentFlow
           acl={this.acl}
           changesFilterByParent={
             changesUtils.filterByParent.bind(changesUtils, currentChangeSet)}
           changeState={this.changeState.bind(this)}
+          cloud={cloud}
+          credential={
+            credentialTag ? credentialTag.replace('cloudcred-', '') : undefined}
```
Comment (hatched) [inline jujugui/static/gui/src/app/app.js]: If you could add a comment here to the effect of "Removing juju tag from credential name....." 

Just so future us knows what's going on here.

**Comment by hatched** [comment]
Thanks @huwshimi Works as expected!
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**